common/app/views/fragments/header.scala.html

```diff
@@ -9,7 +9,7 @@
     @fragments.newHeader()
 } else {
     <header id="header"
-            class="l-header u-cf @if(page.metadata.hasSlimHeader) {l-header--is-slim l-header--no-navigation} js-header"
+            class="l-header u-cf @if(page.metadata.hasSlimHeader) {l-header--is-slim l-header--no-navigation} @if(mvt.ABGalleryRedesignVariant.isParticipating){l-header--is-slim l-header--no-navigation l-header--new-gallery} js-header"
```

Comment:
Does this mean you'll get the slim header if you're in the test but not on a gallery?